packages/grafana-data/src/types/plugin.ts

```diff
@@ -16,11 +16,23 @@ export enum PluginType {
 }
 
 export enum PluginSignatureStatus {
+  unsigned = 'unsigned', // no MANIFEST file
   internal = 'internal', // core plugin, no signature
   valid = 'valid', // signed and accurate MANIFEST
   invalid = 'invalid', // invalid signature
   modified = 'modified', // valid signature, but content mismatch
-  unsigned = 'unsigned', // no MANIFEST file
+  missing = 'missing', // missing signature file
+}
+
+export enum PluginErrorCode {
```

Comment:
Is it confusing having signature related statuses in both `PluginSignatureStatus` and `PluginErrorCode`? Should we maybe rename `PluginErrorCode` as `BackendPluginErrorCode` instead?

Reply:
My thinking is that `PluginErrorCode` may represent other types of plugin errors, not only the signature ones